webtau-docs/webtau/REST/JSON-schema.md

```diff
@@ -0,0 +1,32 @@
+# Validation
+
+Webtau supports validation of objects against [JSON Schema](https://json-schema.org/).  For example, to validate the
+entire response body use the `complyWithSchema` matcher as follows:
+
+:include-groovy: scenarios/rest/jsonSchema/validateSchema.groovy {bodyOnly: true, title: "Example schema validation"}
```

Comment:
will it make sense to include an example where the not the entire body is validated?